### DISCUSSION TITLE
光映广场电影院门口捡到一枚徽章

### DISCUSSION THREAD

**Discussion by McQueen979** [share121/inter-knot#76] [其他]
耳朵凑近听到一声“……姐……我……不想……” 是什么意思呀？
![13帕朵 空梦](https://github.com/user-attachments/assets/b0bfa818-7667-481b-8a7e-99e3dedccc94)

**Comment by Aixi-er** [comment]
啊米诺斯

**Comment by Luli-Chen** [comment]
光顾着刷新，忘记刷你了

**Comment by Tragedysaver1024** [comment]
这位先生你好，这枚徽章有可能是以骸受害者留下的遗物，建议您现在立刻将此徽章埋入土中，在填埋时请务必小声地说出“安息吧，帕朵”。
请相信我。

**Comment by McQueen979** [comment]
我知道了。我相信她一定是一位平凡而又伟大的人。

**Comment by RS64671012** [comment]
> 光顾着刷新，忘记刷你了

公式做题就是快

**Comment by huang8888888** [comment]
芽衣姐，我.我不想死

**Comment by wanwuzhishi** [comment]
好家伙，

**Comment by McQueen979** [comment]
怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子

**Reply by Anonymous755** [reply]
![Screenshot_20240809_200856](https://github.com/user-attachments/assets/91b3cd73-8498-410b-86f0-83c600acf82b)

**Reply by Tragedysaver1024** [reply]
> 怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子

点一下你要回复的帖子右上角的三个点，再点一下Quote Reply

**Comment by Tragedysaver1024** [comment]
> 怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子

点一下你要回复的帖子右上角的三个点，再点一下Quote Reply

**Reply by LB614-111** [reply]
> > 怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子
> 
> 点一下你要回复的帖子右上角的三个点，再点一下Quote Reply

学会了学会了

**Comment by Tragedysaver1024** [comment]
> > > 怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子
> > 
> > 
> > 点一下你要回复的帖子右上角的三个点，再点一下Quote Reply
> 
> 学会了学会了

你这不是没学会吗，耍我啊混账（恼）

**Reply by StarrySkyehk** [reply]
> > > > 怎么回复别人呀？我之前回复别人的回复不显示在绳网上，所以只能评论本帖子
> > > 
> > > 
> > > 点一下你要回复的帖子右上角的三个点，再点一下Quote Reply
> > 
> > 
> > 学会了学会了
> 
> 你这不是没学会吗，耍我啊混账（恼）
>hh

**Comment by zhangyj0523** [comment]
hh，楼上绳网新手好好笑

**Comment by FuningnaFurina** [comment]
不是泥
